### ISSUE TITLE
Initramfs is not working after install

### ISSUE THREAD

**Issue by lmaudio** [Nerun/remastersys#14]
The refreshing of the initramfs is not working on a livesystem remastered with remastersys-gui 4.8/4.9 after install and has to be repaired.

Giving the error code of " missing Command" while initramfs-tools are installed.
Is this just on my machine or reproducable!?

**Comment by nerun** [comment]
Please, give me more details:
- System you are using?
- What have you tried to do? 
- What the exact commands have you typed?
- log file maybe (/home/remastersys/remastersys.log).

**Comment by lmaudio** [comment]
Yes, you're right! It seems to be that it belongs maybe to some differences between Debian and Linux Mint!? I just made a remastersys (backupmode) of a clean and customized Linux Mint 21.1 with Mate desktop. Using remastersys 4.8 because 4.9 gave error messages because of missing gubfiles in remastersys folder. Also 4.8 gave a stable livesystem that is able to be installed. 

Everything works fine after installiert, just the initramfs not. Command of sudo update-initramfs -u -k all gaves the error message: 'command not found'..

Maybe this helps you understanding the problen: 

If i do:

sudo apt purge remastersys remastersys-gui
sudo apt autopurge
sudo apt autoremove
sudo apt autoclean
sudo apt clean
sudo rm -f /bin/remastersys* /bin/ysudo
sudo rm -rf /etc/remastersys
sudo rm -f /etc/remastersys.conf
sudo rm -rf /lib/remastersys
sudo rm -rf /usr/share/doc/remastersys
sudo rm -rf /usr/share/doc/remastersys-gui
sudo rm -f /usr/share/menu/remastersys
sudo rm -f /usr/share/applications/remastersys*

everything is going fixed and initramfs and commands are found after this!

Maybe an uncompability with debiansettings and Linux Mint in live-system or grub settings?

Would be great if remastersys is getting compatible to Linux Mint (again) or that somebody maybe maintain a special Version for this!?  

Also i found a fix with purging remastersys after system install and got a stable image that is really usable.

**Comment by lmaudio** [comment]
I will try isolinux instead oft grub now!
Maybe the problem will be solved this way?!?

I noticed that there are coming no xorriso-dependencies if i install Version 4.9 of remastersys and had xorriso NOT installed before. It is not longer needed in remastersys??

**Comment by nerun** [comment]
Maybe you have genisoimage installed. In dependencies they are written this way: `xorriso | genisoimage`. The pipe "|" means "or". So, maybe, if you have genisoimage installed on your system, when you install remastersys, dependencies see you have genisoimage and abort installation of xorriso.

Xorriso is a MUST HAVE dependecy, you need it for GRUB. Xorriso is optional just for ISOLINUX boot. ISOLINUX will use xorriso by default, but if it don't find xorriso it will use genisoimage. But dor GRUB IT USES grub-mkrescue that DOES NEED xorriso.

So, you should install xorriso: `sudo apt install xorriso`. You can try to fix dependencies too: `suso apt install -f`.

**Comment by nerun** [comment]
Thank you for all your help here. If wasn't you and the bugs you have found i would not fix so many bugs so fast.

**Comment by lmaudio** [comment]
Yea great. Thank you sooo much too.

Yesterday i made an image with ISOLINUX and remastersys 4.9 without any errors. But i had no xorriso installed! So it is wrong now because grub will something missing? Or is it OK?

Genisoimage made a hybrid livestick.

**Comment by nerun** [comment]
ISOLINUX uses xorriso or genisoimage (if xorriso is missing). But GRUB will use just xorriso.

> So it is wrong now because grub will something missing? Or is it OK?

It's all right.

**Comment by lmaudio** [comment]
Yea 
🐰

**Comment by lmaudio** [comment]
Comment moved to issue #16 by @nerun.
